tests/formats/pdb.cpp

```diff
@@ -127,14 +127,17 @@ TEST_CASE("Read files in PDB format"){
 TEST_CASE("Write files in PDB format") {
     auto tmpfile = NamedTempPath(".pdb");
     const auto EXPECTED_CONTENT =
+    "MODEL    1\n"
     "CRYST1   22.000   22.000   22.000  90.00  90.00  90.00 P 1           1\n"
     "ATOM      1 A    XXX X   1       1.000   2.000   3.000  1.00  0.00           A\n"
     "HETATM    2 B    XXX X   2       1.000   2.000   3.000  1.00  0.00           B\n"
     "ATOM      3 C    XXX X   3       1.000   2.000   3.000  1.00  0.00           C\n"
     "HETATM    4 D    XXX X   4       1.000   2.000   3.000  1.00  0.00           D\n"
     "CONECT    1    2\n"
     "CONECT    2    1\n"
+    "ENDMDL\n"
     "END\n"
```

Comment:
Why are there two END in this file ?

Reply:
Because of how is written:
```
    {
        auto file = Trajectory(tmpfile, 'w');
        file.write(frame);
    }
```
`file` is going out of scope twice.

Reply:
Ok, thanks!